### ISSUE TITLE
Option to enable 444 on default site and access list

### ISSUE THREAD

**Issue by axiom00** [NginxProxyManager/nginx-proxy-manager#622]
Would it be possible to add a fifth option on the default site configuration to return 444 connection dropped? Also nice to have the 444 option as a toggle in the access list Deny All rule instead of the current 403.

Thanks.

**Comment by brtwrst** [comment]
I have the same "Issue"/Wish.
You can work around it a little bit.
1. Go to settings and select `404 Page` as Default Site
2. SSH into your machine and go to `[npm_data_volume]/nginx/default_host/site.conf` and change the 404 to 444 in that file.

This change will persist until you select a new option for the Default Site.

**Comment by webartifex** [comment]
Just saw this issue and want to encourage that the above suggestions become features eventually.

I work with the exact same fixes, but having them as GUI options for not so "tech" savvy people would probably be great.

**Comment by LaazV** [comment]
Yes, please! Been wanting this for a while since I'm using a wildcard dns entry, and don't want the server responding to any subdomain.

**Comment by KGBist2000** [comment]
yes please, return code 444 is much safer, than 404. 
One of the usecase is when somebody try to connect via IP and not via hostname/domainname to close the connection by 444 and do not return any information about the server.

**Comment by tomassj9** [comment]
Up for this. Custom HTTP codes are an improvement in my opinion.

**Comment by frenzis01** [comment]
If i got it right this works for the default page, is there any workaround to avoid responding at all (yielding 444 instead of 403) when the client does not match the access list?

**Comment by webartifex** [comment]
Still waiting for it :)

**Comment by sym0nd0** [comment]
> Still waiting for it :)

You'd probably be better to migrate over to [NPMplus](https://github.com/ZoeyVid/NPMplus) and request this change there, as this project has been pretty much abandoned for years. NPMplus is under active development and has a lot of QoL features missing from the original it was forked from.
